bundles/org.eclipse.swt/pom.xml

```diff
@@ -57,4 +57,77 @@
         </plugins>
     </build>
 
+    <profiles>
```

Comment:
R4_30 doesn't have these changes - https://github.com/Halliburton-Landmark/eclipse.platform.swt/blob/R4_30/bundles/org.eclipse.swt/pom.xml

We didn't do such customizations either.